src/commands/run.rs

```diff
@@ -300,7 +300,14 @@ pub async fn run(args: Args) -> Result<()> {
         })?;
     }
 
-    let inferred = Inputs::coalesce(args.inputs)?.into_engine_inputs(document)?;
+    let inferred = Inputs::coalesce(&args.inputs)
+        .with_context(|| {
+            format!(
+                "failed to parse inputs from `{sources}`",
```

Comment:
We should have each source be wrapped in backticks instead of all sources wrapped in one pair of backticks.

Reply:
it does that.

```sh
➜  sprocket git:(fix/input-json-error) ✗ cargo r -- run https://raw.githubusercontent.com/stjudecloud/workflows/refs/heads/ci/docs/workflows/reference/qc-reference.wdl test_jsons/qc-ref-hg19.json test_jsons/qc-ref-hg19.json
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.12s
     Running `target/debug/sprocket run 'https://raw.githubusercontent.com/stjudecloud/workflows/refs/heads/ci/docs/workflows/reference/qc-reference.wdl' test_jsons/qc-ref-hg19.json test_jsons/qc-ref-hg19.json`
error: failed to parse inputs from `test_jsons/qc-ref-hg19.json`, `test_jsons/qc-ref-hg19.json`

Caused by:
    expected `,` or `}` at line 2 column 40
```

Reply:
Oh! Okay I misread it then.